src/riak_object.erl

```diff
@@ -419,7 +422,8 @@ fold_contents({r_content, Dict, Value}=C0, MergeAcc, Clock) ->
                     %% value that is the result of a PUT that
                     %% reconciled this sibling value, we can therefore
                     %% (potentialy, see above) drop this value.
-                    MergeAcc#merge_acc{drop=add_drop_candidate(PureDot, C0, Drop)};
+                    MergeAcc#merge_acc{drop=add_drop_candidate(PureDot, C0,
```

Comment:
there are loads of these formatting changes in all the file diffs. Is it some auto thing emacs is doing on save? I know that the changes make the file better, but they confuse the review. I'd rather just look at relevant changes. If I have to skip over changes because they're just formatting then I may skip over something of value. Please in future, unless you have a string reason, don't mix formatting and functional changes. You can do a new PR/diff after the functional changes with the clean up if you like.